app/src/features/projects/view/components/TeamMember.tsx

```diff
@@ -0,0 +1,107 @@
+
+  // Project data must be loaded by a parent before this component is rendered
+  assert(projectContext.projectDataLoader.data);
+
+  // Define a custom sorting order for roles
+  const roleOrder: { [key: string]: number } = {
+    Coordinator: 1,
+    Collaborator: 2,
+    Observer: 3
+  };
+
+  const projectTeamMembers: IProjectParticipantsRoles[] = useMemo(
+    () =>
+      projectContext.projectDataLoader.data?.projectData.participants
+        .map((member) => {
+          const initials = member.display_name
+            .split(',')
+            .map((name) => name.trim().slice(0, 1).toUpperCase())
+            .reverse()
+            .join('');
+          return {
+            display_name: member.display_name,
+            roles: member.project_role_names,
+            avatarColor: getRandomHexColor(member.system_user_id),
+            initials
+          };
+        })
+        .sort((a, b) => {
+          const roleA = a.roles[0] || '';
+          const roleB = b.roles[0] || '';
+          return roleOrder[roleA] - roleOrder[roleB];
+        }) || [],
+    [projectContext.projectDataLoader.data.projectData.participants]
+  );
+
+  function getRandomHexColor(seed: number) {
+    // Define a seeded random number generator
+    const seededRandom = (min: number, max: number) => {
+      const x = Math.sin(seed++) * 10000;
+      return Math.floor((x - Math.floor(x)) * (max - min + 1)) + min;
+    };
+
+    // Generate a random color component and convert it to a two-digit hex value
+    const randomComponent = () => seededRandom(70, 150).toString(16).padStart(2, '0');
+
+    // Concatenate three random color components to form a hex color
+    return `#${randomComponent()}${randomComponent()}${randomComponent()}`;
+  }
+
+  return (
+    <Stack spacing={1}>
+      {projectTeamMembers.map((member, index) => (
+        <Box display="flex" alignItems="center" key={member.display_name}>
+          <Box
+            sx={{ height: '35px', width: '35px', minWidth: '35px', borderRadius: '50%' }}
+            bgcolor={member.avatarColor}
+            display="flex"
+            alignItems="center"
+            justifyContent="center"
+            mr={1}>
+            <Typography sx={{ fontSize: '0.9rem', color: '#fff' }} variant="body2">
+              {member.initials}
+            </Typography>
+          </Box>
+          <Typography variant="body2" color="textSecondary" alignItems="center" display="flex">
+            {member.display_name}
+            {member.roles.includes('Coordinator') && (
+              <Icon
+                title={member.roles.join(', ')}
+                color="gray"
+                path={mdiCrown}
+                size={0.75}
+                style={{ marginLeft: '8px' }}
+              />
+            )}
+            {member.roles.includes('Collaborator') && (
+              <Icon
+                title={member.roles.join(', ')}
+                color="gray"
+                path={mdiPencilOutline}
```

Comment:
We use the pencil icon in a few places to indicate that we are can make edits, so having that icon next to the user names made me think I could click and edit the users. I'd suggest maybe the account-group icon? it looks like it might fit with the 'Collaborator'